src/OSPSuite.Core/Domain/Quantity.cs

```diff
@@ -38,6 +40,12 @@ public interface IQuantity : IFormulaUsable, IUsingFormula, IWithValueOrigin
       /// </summary>
       double ValueInDisplayUnit { get; set; }
 
+      /// <summary>
+      ///    The value in the displayed unit
+      /// </summary>
+      bool TryGetValueInDisplayUnit(out double result);
```

Comment:
Should `TryGetValueInDisplayUnit `be used **only** for tests? Then: why not defining it as Extension Method?

Reply:
No not for test. I want to use it in real code when the debugger is running, catching the error all the time creates a lot of overhead. It makes working with the tool while debugging very hard (initial concentration formula are all invalid and the whole UI freezes). This is a problem in debug mode only that I wanted to fix for a while. Now the pain is too big so it's required

Reply:
But we can define it as extension method. I don't see a problem

Reply:
This will be used in ParameterDTO in pksim

Reply:
ok, if not for tests onle, then it's OK I think